docs/self-hosting/methods/install-methods-commercial/kubernetes.md

```
@@ -0,0 +1,568 @@
+
+| Host | Path | Service |
+|---    |:---:|---|
+| plane.example.com | `/` | `<http://plane-app-web.plane:3000>` |
+| plane.example.com | `/spaces/*`  | `<http://plane-app-space.plane:3000>` |
+| plane.example.com | `/god-mode/*` | `<http://plane-app-admin.plane:3000>` |
+| plane.example.com | `/live/*` | `<http://plane-app-live.plane:3000>` |
+| plane.example.com | `/api/*`  |  `<http://plane-app-api.plane:8000>` |
+| plane.example.com | `/auth/*` | `<http://plane-app-api.plane:8000>` |
+| plane.example.com | `/uploads/*` | `<http://plane-app-minio.plane:9000>` |
+| plane-minio.example.com | `/` | `<http://plane-app-minio.plane:9090>` |
+| plane-mq.example.com | `/` | `<http://plane-app-rabbitmq.plane:15672>` | (Optional) if using local setup, this will enable management console access |
```

markdownlint-cli2 (0.20.0)

[warning] Table column count
Expected: 3; Actual: 4; Too many cells, extra data will be missing

(MD056, table-column-count)